roles/db_restore/tasks/restore_database.yml

```diff
@@ -1,5 +1,17 @@
 ---
 
+- name: stop unicorn and delayed_job
+  service:
+    name: "{{ service_name }}"
+    state: stopped
+  loop:
+    - "unicorn_{{ app }}"
+    - "delayed_job_{{ app }}"
+  loop_control:
+    loop_var: service_name
+  become: yes
+  become_user: root
```

Comment:
wouldn't it be safer to become_user: "{{ user }}" instead? and it's probably better to do that in the playbook so the role is not coupled to it.

Reply:
It doesn't work. We get this error: 
```
failed: [local_travis] (item=delayed_job_openfoodnetwork) => {"ansible_loop_var": "service_name", "changed": false, "msg": "Unable to start service delayed_job_openfoodnetwork: Failed to start delayed_job_openfoodnetwork.service: Interactive authentication required.\nSee system logs and 'systemctl status delayed_job_openfoodnetwork.service' for details.\n", "service_name": "delayed_job_openfoodnetwork"}
```
I'm pretty sure that's why I added this in the first place.